dnf/const.py.in

```diff
@@ -25,6 +25,8 @@ CONF_AUTOMATIC_FILENAME='/etc/dnf/automatic.conf'
 DISTROVERPKG=('system-release(releasever)', 'system-release',
               'distribution-release(releasever)', 'distribution-release',
               'redhat-release', 'suse-release')
+DISTROVER_MAJOR_PKG='system-release-major'
+DISTROVER_MINOR_PKG='system-release-minor'
```

Comment:
Would it make more sense to use `system-release(releasever_major)` and `system-release(releasever_minor)` for the provides for consistency with the existing `system-release(releasever)`?

Reply:
Ah, yes, those are better, I changed it. DNF respects either `system-release` or `system-release(releasever)` but looks like `libdnf`'s `dnf_context_set_os_release` only looks at `system-release(releasever)` and not `system-release`.